Core.Collectors/Cache/AzureTableCache.cs

```diff
@@ -3,75 +3,70 @@
 
 using Microsoft.CloudMine.Core.Collectors.IO;
 using Microsoft.CloudMine.Core.Collectors.Telemetry;
-using Microsoft.WindowsAzure.Storage;
-using Microsoft.WindowsAzure.Storage.Table;
 using System;
 using System.Collections.Generic;
+using System.Net;
 using System.Threading.Tasks;
+using Azure;
+using Azure.Data.Tables;
 
 namespace Microsoft.CloudMine.Core.Collectors.Cache
 {
-    public class AzureTableCache<T> : ICache<T> where T : TableEntityWithContext
+    public class AzureTableCache<T> : ICache<T> where T : TableEntityWithContext, new()
     {
         private readonly ITelemetryClient telemetryClient;
         private readonly string name;
         private readonly string storageConnectionEnvironmentVariable;
 
-        private CloudTable table;
+        private TableClient table;
         private bool initialized;
 
         public AzureTableCache(ITelemetryClient telemetryClient, string name, string storageConnectionEnvironmentVariable = "AzureWebJobsStorage")
         {
             this.telemetryClient = telemetryClient;
-            this.initialized = false;
```

Comment:
Curiosity question:
why remove the `this.` instance references here?

Reply:
It doesn't hurt anything, but it also doesn't provide any benefit since there is nothing else with the same name to disambiguate it from.